File: WS_EUREKABANK_SOAPDOTNET_GR04/01.SERVIDOR/WS_EB_DOTNET_SOAP_Servidor/ec.edu.monster.modelo/Movimiento.cs
using System;
using System.Xml.Serialization;

namespace WS_EB_DOTNET_SOAP_Servidor.ec.edu.monster.modelo
{
    [Serializable]
    [XmlType("movimiento")]
    public class Movimiento
    {
        [XmlElement("cuenta")]
        public string Cuenta { get; set; }

        [XmlElement("nromov")]
        public int NroMov { get; set; }

        [XmlElement("fecha", DataType = "dateTime")]
        public DateTime Fecha { get; set; }

        [XmlElement("tipo")]
        public string Tipo { get; set; }

        [XmlElement("accion")]
        public string Accion { get; set; }

        [XmlElement("importe")]
        public decimal Importe { get; set; } // dinero -> decimal
    }
}

File: WS_EUREKABANK_SOAPJAVA_GR04/02.CLIESC/CLIESC_EurekaBank_SOAP_GR04/src/main/java/ec/edu/monster/model/Movimiento.java
package ec.edu.monster.model;

import jakarta.xml.bind.annotation.XmlAccessType;
import jakarta.xml.bind.annotation.XmlAccessorType;
import jakarta.xml.bind.annotation.XmlRootElement;
import java.util.Date;

@XmlRootElement(name = "movimiento")
@XmlAccessorType(XmlAccessType.FIELD)
public class Movimiento {
  private String cuenta;
  private int nromov;
  private Date fecha;
  private String tipo;
  private String accion;
  private double importe;

  public String getCuenta() { return cuenta; }
  public void setCuenta(String cuenta) { this.cuenta = cuenta; }
  public int getNromov() { return nromov; }
  public void setNromov(int nromov) { this.nromov = nromov; }
  public Date getFecha() { return fecha; }
  public void setFecha(Date fecha) { this.fecha = fecha; }
  public String getTipo() { return tipo; }
  public void setTipo(String tipo) { this.tipo = tipo; }
  public String getAccion() { return accion; }
  public void setAccion(String accion) { this.accion = accion; }
  public double getImporte() { return importe; }
  public void setImporte(double importe) { this.importe = importe; }
}
